README.md

```diff
@@ -23,7 +23,7 @@ Available CDNs are:
 <body>
 <form name="checkoutForm" method="POST" action="test.html">
     <!--Sample checkout for a 995.00 THB charge-->
-    <script type="text/javascript" src="https://cdn.omise.co/card.js.gz"
```

Comment:
Oh !, it because I saw the CDN list in `Installation` section is
- Singapore: `https://cdn.omise.co/card.js`
- Japan: `https://cdn2.omise.co/card.js`

So, the reason that I decided to removed `.gz` is for prevent a confusion that `.gz` was not present in CDN list.

Sorry, I forgot to tell you about this 😅 (do you want me to bring it back?)